diamm/templates/website/organization/organization_detail.jinja2

```diff
@@ -44,6 +44,8 @@
                 {% endfor %}
                 </ul>
             {% endif %}
+            {% set contributions = contribution_sort(content.name) %}
```

Comment:
I'll be in today, so maybe you can explain to me what the contribution_sort method does. I'm not entirely sure why you have it here.